codecov.yml

```
@@ -14,4 +14,12 @@ coverage:
   behavior: default
-  require_changes: yes
\ No newline at end of file
+  require_changes: yes
+
+ignore:
+  - '**/*.pb.go'
+
+fixes:
+  - github.com/go-fries/fries/cache/v3/::cache/
+  - github.com/go-fries/fries/cache/redis/v3/::cache/redis/
+  - github.com/go-fries/fries/chi/v3/::chi/
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)